openreview/venue/process/invitation_edit_process.py

```diff
@@ -133,27 +133,34 @@ def updated_content_readers(note, paper_inv):
 
     def post_invitation(note):
 
-        content = {
-            'noteId': {
-                'value': note.id
-            },
-            'noteNumber': {
-                'value': note.number
+        if isinstance(note, tuple):
```

Comment:
I'm not fully convinced about checking if the note is a tuple or not.

Reply:
We could always return a tuple as [note, forumNote] where in the case of submissions they are the same note:

`source_submissions = [(n, n) for s in client.get_all_notes(content={ 'venueid': venue_id }, sort='number:asc')]`

and when we build the invitation edit content:

```
{
  noteId: forumNote.id,
  noteNumber: forumNote.number
}
 if 'replyto' in invitation.edit['content']:
        content['replyto'] = { 'value': note.id }

....
```

I think `noteId` and `noteNumber` should have been called `forumId` and `forumNumber` but I think it is too late.